pages/accordion_page.ts

```diff
@@ -0,0 +1,26 @@
+import { Page, expect, test } from '@playwright/test';
+
+export class AccordionPage {
+    readonly page: Page;
+
+    constructor(page: Page) {
+        this.page = page;
+    }
+
+    async open() {
+        await this.page.goto('https://webdriveruniversity.com/Accordion/index.html');
+    }
+
+    async waitForLoadedComplete() {
+        await test.step ('Wait for complete loading.', async () => {
+            await expect(this.page.locator('#hidden-text')).toHaveText('LOADING COMPLETE.',{timeout: 20000});
```

Comment:
You should create elements in constructor. What if I want use page.locator('#hidden-text') in other place?